x-pack/plugins/security_solution/public/timelines/store/defaults.ts

```diff
@@ -33,7 +33,6 @@ export const timelineDefaults: SubsetTimelineModel &
   description: '',
   eqlOptions: {
     eventCategoryField: 'event.category',
-    tiebreakerField: '',
```

Comment:
`tiebreakerField` as an empty doesn't look correct for the EQL validator. It sends requests to `/internal/search/eql` to validate the request. It returns an error when a tiebreaker field has an empty string.

Additionally it looks like EQL options weren't provided correctly before this PR. Timeline EQL validation requests were sent without EQL options.